src/stylesheets/mixins.scss

```diff
@@ -0,0 +1,13 @@
+@mixin background-cover($position) {
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: $position;
+}
+
+@mixin position($position, $top: null, $right: null, $bottom: null, $left: null) {
```

Comment:
This mixin does not save a lot of typing, but helps to track where absolute positioning is used for later refactoring